ndio/remote/neurodata.py

```diff
@@ -22,9 +22,9 @@
 except ImportError:
     import urllib2
 
-DEFAULT_HOSTNAME = "openconnecto.me"
+DEFAULT_HOSTNAME = "cloud.neurodata.io"
```

Comment:
For now we should not change this, essentially because some datasets are stored in different places and only openconnecto.me has the load-balancer for distribution. Additionally when you come in today we'll talk about setting up a vm on your local computer and running everything against that. As a general note, any code development should never be done against the production servers (that includes testing).